Prime Checker/prime checker c.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+
+int prime(int n)
+{
+	int i;
+  	if (n == 1) 
+	{
+    		return 0;
+  	}
+  	else 
+	{
+    		for (i = 2; i < n; i++) 
+		{
+      			if (n % i == 0)

```

Comment:
Stills miss indented, but you are closer! Pay attention and format it again, you can do it! ✨

Reply:
done